inference-engine/src/legacy_api/src/ngraph_ops/prior_box_clustered_ie.cpp

```diff
@@ -44,15 +44,6 @@ bool op::PriorBoxClusteredIE::visit_attributes(AttributeVisitor& visitor) {
     visitor.on_attribute("step", step);
     visitor.on_attribute("step_w", m_attrs.step_widths);
     visitor.on_attribute("step_h", m_attrs.step_heights);
-    if (step != 0) {
```

Comment:
If I remember right we added this code because we had some issues with serialization/deserialization. Are you sure that we can remove this code without any degradations?

Reply:
PriorBoxClusteredIE is being created from PriorBoxClustered attributes in convert_prior_to_ie_prior.cpp and there is no
'step' attribute so in the end in the legacy version 'step' always ends up as a 0 so it seems like a dead code from my point of view.